No rule to make target 'Ref_SignalGen_testimpl' with fprime-util impl --ut in tutorial

| | |
|:---|:---|
|**_F´ Version_**| 85fa25b |
|**_Affected Component_**| fprime-util, tutorial  |
---
## Problem Description
`fprime-util impl --ut` fails on vanilla tutorial installation:
```
(fprime-venv) user@computer /localdata2/user/fprime1/fprime/Ref/SignalGen % fprime-util impl --ut
[WARNING] Failed to find settings file: /localdata2/user/fprime1/fprime/Ref/settings.ini
make: *** No rule to make target 'Ref_SignalGen_testimpl'.  Stop.
[CMAKE] CMake failed to detect target, attempting CMake cache refresh and retry
make: *** No rule to make target 'Ref_SignalGen_testimpl'.  Stop.
[ERROR] CMake erred with return code 2
```


## How to Reproduce

1. Fresh install as per INSTALL.md (functioning HTML GUI)
2. Getting started tutorial as per Tutorial.md, skipping raspberry pi build
```
cd fprime/Ref
fprime-util generate
cd fprime/Ref/SignalGen
fprime-util impl
```
(Runs successfully to 100%, generates -template files)
3. Attempt to generate unit test implementation stubs
```
fprime-util impl --ut
```

## Expected Behavior

Stubbed implementation files for the unit tests are created.

--- a/src/fprime/fbuild/builder.py
+++ b/src/fprime/fbuild/builder.py
@@ -620,7 +620,13 @@ BUILD_TARGETS = [
     ),
     # Implementation targets
     LocalTarget("impl", "Generate implementation template files"),
-    LocalTarget("impl", "Generate unit test files", flags={"ut"}, cmake="testimpl"),
+    LocalTarget(
+        "impl",
+        "Generate unit test files",
+        flags={"ut"},
+        cmake="testimpl",
+        build_type=BuildType.BUILD_TESTING,
+    ),
     # Check targets and unittest targets
     LocalTarget("check", "Run unit tests", build_type=BuildType.BUILD_TESTING),
     LocalTarget(

